### PR TITLE
Added loading indicator to PlayerViewController when context is empty

### PR DESCRIPTION
**Pull request by balavor**
## Issue
While the SDK process the ad for too long, users see a black screen without any indication of activity.

## Changes
- Added loading indicator in `PlayerViewController` that will be displayed when we have `empty` state in our `context presenter` (instead of `ad` or `content`);
- Image for `loadingImageView` is taken from `PlayerControls` bundle.
- Added rotation methods for imageView. 

## Demo
| - | Old | New |
| - | - | - |
| Content | content-old | content-new |
| Ad | ad-old | ad-new |

@OathAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: sources/custom controls/PlayerViewController.swift [hunk 1]
```diff
@@ -69,6 +82,18 @@ public final class PlayerViewController: UIViewController {
             adControls.didMove(toParent: self)
         }
         
+        /* Create -> configure -> add loading indicator to video view */ do {
+            let loadingImage = UIImage(named: "icon-loading", in: Bundle(for: AdVideoControls.self), compatibleWith: view.traitCollection)
+            loadingImageView = UIImageView(image: loadingImage)
+            loadingImageView.center = view.center
+            loadingImageView.autoresizingMask = [.flexibleLeftMargin,
```
Comment (VladyslavAnokhin) [inline sources/custom controls/PlayerViewController.swift]: Why autoresizing masks? 
Let's use auto layouts

File: sources/utils/Animations.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+extension UIImageView {
+    func enableRotation() {
```
Comment (VladyslavAnokhin) [inline sources/utils/Animations.swift]: We already have same code in `source/Utils` =\

File: sources/custom controls/PlayerViewController.swift (hunk 1 above)
Comment (VladyslavAnokhin) [reply sources/custom controls/PlayerViewController.swift]: And maybe make sense to move this loading view into separate view, with two public methods start/stop loading or `show/hideLoadingView(on superView: UIView)`.

File: sources/utils/Animations.swift (hunk 2 above)
Comment (balavor) [reply sources/utils/Animations.swift]: So? I don't want to make them `public` for everyone.

File: sources/custom controls/PlayerViewController.swift (hunk 1 above)
Comment (balavor) [reply sources/custom controls/PlayerViewController.swift]: Not a problem. And those animation methods can be added right for this view

**Comment by balavor** [comment]
I've updated this PR and created a new one in Player Controls. Waiting for your review.

File: sources/custom controls/PlayerViewController.swift [hunk 3]
```diff
@@ -10,6 +10,8 @@ import WebKit
 /// Reacts on events from `Player` and sends action to `Player`.
 public final class PlayerViewController: UIViewController {
     
+    private var loadingImageView: LoadingImageView!
```
Comment (AndriiMoskvin) [inline sources/custom controls/PlayerViewController.swift:13]: Why `!`? Can this be optional?

**Review by AndriiMoskvin — DISMISSED** [review]

File: sources/custom controls/PlayerViewController.swift (hunk 3 above)
Comment (VladyslavAnokhin) [reply sources/custom controls/PlayerViewController.swift:13]: Will be updated in the next PR

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**